### PR TITLE
docs: add types of changes supported to FAQ

### PR DESCRIPTION
**Pull request by bryanoltman**
## Status

**READY**

## Description

Clarifies which types of changes we support and do not support.

Part of https://github.com/shorebirdtech/docs/issues/71

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -70,6 +70,14 @@ guide for [Github Actions](ci/github.md), other CI systems should be similar.
 Please don't hestiate to reach out over Github issues or Discord if you
```
Comment (felangel) [inline docs/faq.md:70]:
```suggestion
Please don't hestiate to reach out over GitHub issues or Discord if you
```

**Review by felangel — APPROVED** [review]

File: docs/faq.md [hunk 2]
```diff
@@ -67,9 +67,17 @@ https://github.com/shorebirdtech/shorebird/issues/739
 Yes. Shorebird is intended to be used from CI systems. We've published a
 guide for [Github Actions](ci/github.md), other CI systems should be similar.
 
-Please don't hestiate to reach out over Github issues or Discord if you
+Please don't hestiate to reach out over GitHub issues or Discord if you
 encounter any issues.
 
+### What types of changes does Shorebird support?
+
+Shorebird can change any Dart code in your application. This includes app code,
```
Comment (nilsreichardt) [inline docs/faq.md:75]: Would it be useful to also clarify that with Dart it also means `pubspec.yaml` and `pubspec.lock` (to make clear that you could also add a Dart only package to your app and Shorebird still updates it)?

Comment (bryanoltman) [reply docs/faq.md]: Updated, thanks!

Comment (nilsreichardt) [reply docs/faq.md]: Cool!

**Pull request merged**